source/usage-and-configuration/swagger/introduction.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Swagger
+---
+
+# Swagger
+
+The theme provides tools to read and parse swagger files and generate documentations out of it.
+
+## Tags
+It provides following two tags which can grab the swagger schema from local or a remote location and create beautiful and easy to read documents.
+* **[swagger_ui](./swagger-ui.html) -**  This tag generates [swagger ui](https://swagger.io/swagger-ui) from the swagger schema.
+* **[swagger_to_html](./swagger-to-html.html) -**  This tag generates a simple and clean user interface that contains all the useful information present in the swagger specification with examples.
+
+
+## Decorators
+The theme also provides features to decorate/modify the swagger spec before using it in the theme.
```

Comment:
...The theme also provides features to decorate/modify the swagger spec before using it. (in in the theme is redundant)

Reply:
👍